internal/iso/iso.go

```diff
@@ -122,60 +117,82 @@ func (h *Handler) RoundTrip(req *http.Request) (*http.Response, error) {
 	// RoundTripper needs a Transport to execute a HTTP transaction
 	// For our use case the default transport will suffice.
 	resp, err := http.DefaultTransport.RoundTrip(req)
-	// resp, err := h.RoundTripper.RoundTrip(req)
 	if err != nil {
-		h.Logger.Info("HTTP request didn't receive a response", "sourceIso", h.SourceISO, "error", err)
+		log.Info("issue getting the source ISO", "sourceIso", h.SourceISO, "error", err)
 		return nil, err
 	}
+	if resp.StatusCode != http.StatusOK && resp.StatusCode != http.StatusPartialContent {
+		log.Info("the request to get the source ISO returned a status other than ok (200)", "sourceIso", h.SourceISO, "status", resp.Status)
```

Comment:
are we skipping 206 in the logging line, because we want to abstract out the implementation details of the client and provide context only when necessary?